cmd/cloud/migration.go

```diff
@@ -0,0 +1,41 @@
+package cloud
+
+import (
+	"errors"
+
+	astrocore "github.com/astronomer/astro-cli/astro-client-core"
+	"github.com/astronomer/astro-cli/cloud/organization"
+	"github.com/astronomer/astro-cli/context"
+)
+
+var ErrorShortName = errors.New("cannot find organization shortname")
+
+// migrate config
+func migrateCloudConfig(coreClient astrocore.CoreClient) error {
+	ctx, err := context.GetCurrentContext()
+	if err != nil {
+		return err
+	}
+	// backfill OrganizationShortName
+	if (ctx.OrganizationShortName == "") && (ctx.Organization != "") {
```

Comment:
here is the code that backfill organization_short_name in config

Reply:
just curious why would `organization_short_name` not be filled in because we already call `c.SetOrganizationSetup()` with it if the user comes in via [checkUserSession() or checkToken()](https://github.com/astronomer/astro-cli/pull/873/files#diff-69d508fdcd15361e006cbb3a7f512757cfafb962e5e3ccad1d7071033bb8b713R325), or via [checkAPIKeys()](https://github.com/astronomer/astro-cli/pull/873/files#diff-d961dc2a8e5c959835ea7f2bd6dbb204a39431e38091c81df1bc813bc14164ebR303)

Reply:
because the code in those place are optional. 
1. checkAPIKeys will only run in the context of api key.
2. checkToken will not backfill if the token is valid as we only backfill short name in login path.

so a "catch all" migration step feels safer. I think in the future we will run into migration scenario as well so having a migration step will be helpful.